hyperjumptech/monika
# fixed(docker): add dockerignore to reduce size

# Monika PR
fix #204 reduce docker size
## What issue this PR is trying to solve
Currently the docker image size is around 300MB that was considered large.
## How this PR solve the issue 
Adding node_modules to dockerignore to reduce the docker image size
## How to test
1. Run docker build to build docker image with dockerignore
2. Check docker image size, it should be reduced by 40MB
